WordPress/Classes/Services/PostService.swift

```diff
@@ -0,0 +1,673 @@
+import Foundation
+
+// manages the creation, saving, updating, and merging of Posts, Pages, and associated data such as comments
+class PostService : LocalCoreDataService {
+    static let ErrorDomain = "PostServiceErrorDomain"
+    static let DefaultNumberToSync: UInt = 40
+
+    enum PostType: String {
+        case any = "any"
+        case post = "post"
+        case page = "page"
```

Comment:
(AFAIK) Enum Cases should be uppercase. @diegoreymendez please, correct me if not, because otherwise i'd need to update few things all over!